### PR TITLE
(GH-94) Add check for .DS_Store file

### LINKED ISSUE #94: Requirement: Make sure no .DS_Store files are included in packages
Reference: https://en.wikipedia.org/wiki/.DS_Store

@ferventcoder what are your thoughts on the level this should be at?

### PR DESCRIPTION
**Pull request by gep13**
Closes #94

### CODE REVIEW

**Comment by gep13** [comment]
@ferventcoder didn't see the point in creating a separate rule for this, so just added it to the existing one.

Thoughts?

**Comment by ferventcoder** [comment]
It's not source control, so it may confuse people. It's like a .ThumbsDB file. Which would go along nicely with .DS_Store. Let's create a separate rule though.

**Comment by gep13** [comment]
@ferventcoder let me know what you think about the alteration to this PR.

File: src/chocolatey.package.validator/infrastructure.app/rules/OperatingSystemIndexFilesAreNotPackagedRequirement.cs [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+            }
+        }
+
+        public override PackageValidationOutput is_valid(IPackage package)
+        {
+            var files = package.GetFiles().or_empty_list_if_null();
+            return !files.Any(
+                f => f.Path.to_lower().Contains("thumbs.db")
+                     || f.Path.to_lower().Contains(".ds_store")
+                        );
```
Comment (ferventcoder) [inline src/chocolatey.package.validator/infrastructure.app/rules/OperatingSystemIndexFilesAreNotPackagedRequirement.cs]: :+1:

**Pull request merged**